app/serializers/disability_claim_base_serializer.rb

```diff
@@ -22,11 +22,11 @@ def phase_change_date
   end
 
   def open
-    object.data['claimCompleteDate'].blank?
+    object_data['claimCompleteDate'].blank?
```

Comment:
This class should have `object_data` defined on it, even if it just returns an empty hash. You could have the default be the list data, and the detail serializer uses the data column, but I like having explicit serializers for the different views.